app/controllers/describe_unknown_repair_controller.rb

```diff
@@ -1,18 +1,25 @@
 class DescribeUnknownRepairController < ApplicationController
   def index
+    @back = Back.new(controller_name: 'questions/start')
```

Comment:
As discussed, this `Back` shouldn't be here.

Reply:
I have eliminated it from history like a time travel B-movie